.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "coraza"]
```

Comment:
Submodules tend to be pretty annoying starting with needing recursive clone. Since it's only used during generation, it probably doesn't need to be statically available. If the script accepts a hash (or maybe just automatically use latest and can be a nightly job that creates a PR automatically when there are changes) it can download the code with HTTP.

Reply:
Sounds good. Let me try that.

Reply:
BTW while a different repo it'll be similar to this

https://github.com/corazawaf/coraza/blob/v3/dev/testing/coreruleset/coreruleset_test.go#L44

Reply:
Changed to use `go mod vendor` which I think is the easier as recognizes coraza as a dependency.